src/modules/billing/controllers/batches.js

```diff
@@ -110,9 +110,21 @@ const getBatchInvoiceDetail = async request => {
 const deleteBatchInvoice = async (request, h) => {
   const { batch } = request.pre;
   const { invoiceId } = request.params;
+
+  let originalInvoiceId = null;
+  let rebillInvoiceId = null;
+
+  if (request.query) {
+    originalInvoiceId = request.query.originalInvoiceId;
+    rebillInvoiceId = request.query.rebillInvoiceId;
+  }
```

Comment:
This could be written as 
```const { originalInvoiceId, rebillInvoiceId  } = request.query;```

and then you check if there are values for these on line 122